### PR TITLE
add node modules setup to .bootstrap

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add node modules setup to bootstrap so README validation will work out of the box.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .hooks/bootstrap [hunk 1]
```diff
@@ -130,3 +134,18 @@ else
     fi
 fi
 
+echo "======================="
+
+if [ -n "$NO_NPM" ]; then
+    echo "Skipping npm node modules setup as environment varialbe NO_NPM is set"
+else
+    if ! command -v npm >/dev/null 2>&1; then
```
Comment (anara123) [inline .hooks/bootstrap:142]: you can add this code in update flow too, in case user already have hooks and he runs ./hooks/bootstrap -u

**Review by anara123 — APPROVED** [review]

File: .hooks/bootstrap (hunk 1 above)
Comment (glicht) [reply .hooks/bootstrap:142]: It's there. At line 38. We just check if the node_modules dir exists as an indication.

Comment (anara123) [reply .hooks/bootstrap:142]: but if all the developers that didn't install node_modules and have bootstrap installed, when they will run `.hooks/bootstrap -u` , the process will try to find `node_modules` and will not find it.
It is not a big deal, developers can delete the .venv and install the hook again

Comment (glicht) [reply .hooks/bootstrap:142]: It is better to do accordingly to the folder. So if someone choose to not install we don't do this every time. Devs can run nom install if needed.

**Pull request merged**